aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -576,6 +577,18 @@ Tensor _new_zeros_with_same_meta_batching_rule(
   return self_physical_view.getPhysicalToLogicalMap().apply(result);
 }
 
+Tensor _make_dual_batching_rule(
+  c10::DispatchKeySet ks,
+  const Tensor& primal,
+  const Tensor& tangent,
+  int64_t level
+) {
+  DispatchKeySet after_batched_keyset =
+      DispatchKeySet(DispatchKeySet::FULL_AFTER, c10::DispatchKey::Batched);
+  return at::redispatch::_make_dual(ks & after_batched_keyset, primal, tangent, level);
+}
```

Comment:
What I wanted to do here is to basically say that _make_dual is composite wrt the Batched key, so that we can redispatch to the autograd key first. Maybe there is a better way @zou3519?

Reply:
Can you register a make fallthrough to do that?

Reply:
I'm confused: why doesn't `m.impl("_make_dual", native::_make_dual);` work?

Reply:
If we had `native::_make_dual`, it would go directly to our implementation (which all it does is call `primal.view(primal.shape())`. We'd like for it to go through our kernel in VariableTypeManual, which does other necessary stuff.